Llk/Parser.php

```diff
@@ -192,7 +192,7 @@ public function parse($text, $rule = null, $tree = true)
                 $column = 1;
 
                 if (!empty($text)) {
-                    if (0 === $offset) {
+                    if (0 === $offset || null === $offset) {
```

Comment:
@Hywan offset really equal to null. The problem can be reproduced if the first token and/or character does not correspond to any rule (rule == defined token).